src/tls/maybe_tls.rs

```diff
@@ -54,47 +60,76 @@ impl<R: Clone, T: Clone> Clone for MaybeTls<R, T> {
 }
 
 // Conditionally implement Debug for Debugable types
-impl<R: Debug, T: Debug> Debug for MaybeTls<R, T> {
-    fn fmt(&self, fmt: &mut Formatter) -> fmt::Result {
+impl<R: fmt::Debug, T: fmt::Debug> fmt::Debug for MaybeTls<R, T> {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
I'm not sure why to add the `fmt::` prefix to these.

Reply:
Just removed extra imports (Debug/Formatter) and changed to familiar Debug trait impl. After this PR all Debug impls in codebase will looks the same. But not a big deal for me can change back.